docs/nightly/en/user-guide/migrate-to-greptimedb/migrate-from-influxdb.md

```diff
@@ -0,0 +1,537 @@
+# Migrate from InfluxDB
+
+GreptimeDB is a high-performance time series database designed to work in the infrastructure of the cloud era.
+Users can benefit from its elasticity and commodity storage.
```

Comment:
Consider removing this from the migration guide. We will put together "why greptime" in a dedicated docs.